app/assets/javascripts/client/inventories/MessageElementCtrl.js

```diff
@@ -6,12 +6,13 @@
 
   function MessageElementCtrl() {
     var vm = this;
-    vm.messageTitle = function(category) {
+    vm.messageTitle = function(category, kind) {
+      var subjectKind = kind || 'Inventory';
```

Comment:
It feels like you should pass this in from the directive instead.  The functionality would still be the same, but you'd be getting the `subjectKind` from `$scope`.